samples/custom-voice/python/customvoice/config.py

```diff
@@ -7,7 +7,7 @@
 import logging
 
 class Config:
-    api_version = 'api-version=2023-12-01-preview'
+    api_version = 'api-version=2024-02-01-preview'
```

Comment:
This version is in the progress of release. Please merge after release, or revert this change. We'll update this line after release.